### PR TITLE
Remove IRC Channel Mentions

### PR DESCRIPTION
**Pull request by zoracon**
Fixes https://github.com/certbot/certbot/issues/8880
- IRC channel is no more (we now use mattermost)
- Remove references

### CODE REVIEW

File: privacy/index.md [hunk 1]
```diff
@@ -27,8 +27,6 @@ Let’s Encrypt also operates [Community Forums](https://community.letsencrypt.o
 
 Let’s Encrypt may change these policies or providers from time to time. If you have questions about Let’s Encrypt’s privacy practices, please contact them directly at [security@letsencrypt.org](mailto:security@letsencrypt.org).
 
-The Certbot community can be contacted through the Freenode Project’s IRC service, chat.freenode.net, on the #certbot-devel channel. This IRC service is operated by Freenode, which does not publish a privacy policy.
```
Comment (alexzorin) [inline privacy/index.md:30]: There appears to be a changelog in this privacy policy ("Changes to This Privacy Poilicy"). Should the removal of this paragraph change be documented there?

**Review by alexzorin — CHANGES_REQUESTED** [review]
Just one minor fix. 

Also it looks like there is a merge conflict with `.travis.yml`, which got deleted since this PR was opened.

Otherwise, LGTM, thanks.

File: privacy/index.md (hunk 1 above)
Comment (zoracon) [reply privacy/index.md:30]: I can maybe offer a link to the diff change. I will incorporate this feedback today.

Comment (zoracon) [reply privacy/index.md:30]: Apologies on letting this drop off. Added the change

**Comment by zoracon** [comment]
Ack, my accidently readded a file in merge conflict resolution. Will remove

**Review by alexzorin — APPROVED** [review]
Thanks!

**Pull request merged**